### PR TITLE
doc: merge slate documentation with latest change from old documentation

### PR DESCRIPTION
**Pull request by stifoon**
\o/
JIRA: http://jira.canaltp.fr/browse/NAVIO-469

### CODE REVIEW

File: documentation/slate/source/includes/index.md [hunk 1]
```diff
@@ -137,53 +145,73 @@ In this case it needs a region id. This id can the found in the ``regions`` sect
...
+    - <https://api.navitia.io/v1/coverage/fr-idf/networks/network:OIF:439/physical_modes/physical_mode:Metro/lines>
+- By the way, what stuff are close to me?
+    - <https://api.navitia.io/v1/coverage/fr-idf/coords/2.377310;48.847002/places_nearby>
+    - or <https://api.navitia.io/v1/coverage/fr-idf/coords/2.377310;48.847002/lines>
+    - or <https://api.navitia.io/v1/coverage/fr-idf/coords/2.377310;48.847002/stop_schedules>
+    - or ...
+
 
 
 ### What places have a name that start with 'tran'
```
Comment (stifoon) [inline documentation/slate/source/includes/index.md]: in the example below, the input is "eiff" and not "tran"

File: documentation/slate/source/includes/integration.md [hunk 2]
```diff
@@ -1,58 +1,19 @@
 Navitia documentation: v1 interface
 ===================================
 
-Overview
---------
+vocabulary
```
Comment (stifoon) [inline documentation/slate/source/includes/integration.md]: Vocabulary vs vocabulary

File: documentation/slate/source/includes/details.md [hunk 3]
```diff
@@ -7,55 +7,9 @@ Working on a specific region
 
 In the first examples, the positions are given by coordinates and no network is specified.
 
-
 However when no coordinates are provided, you need to provide on what region you want to request.
 
-The list of region covered by *navitia* are available at <http://api.navitia.io/v0/regions.json> . The shape of the region
+The list of region covered by *navitia* are available at <http://api.navitia.io/v1/coverage> . The shape of the region
```
Comment (pbougue) [inline documentation/slate/source/includes/details.md]: The list of region`s` covered `is` available at

File: documentation/slate/source/includes/index.md [hunk 4]
```diff
@@ -78,7 +89,7 @@ The arguments are the following:
 * ``to=-122.402770;37.794682``
 * ``datetime=20140118T0800``
```
Comment (pbougue) [inline documentation/slate/source/includes/index.md]: Not consistent with the example url (and also dates format 5 lines above).
I would rather move date explanations (these 2 lines) under the url, and provide a new resulting url (with a real datetime or a yymmddThhmmss one)

File: documentation/slate/source/includes/integration.md [hunk 5]
```diff
@@ -1,58 +1,19 @@
...
+----------
 
 This document describes how to call navitia via the v1 interface, and
-the returned resources. You can read our lexicon at
+the returned resources. You can read a mobility lexicon at
 <https://github.com/OpenTransport/vocabulary/blob/master/vocabulary.md>
```
Comment (pbougue) [inline documentation/slate/source/includes/integration.md]: Do we want to keep that link? As we have a vocabulary section in that doc...

File: documentation/slate/source/includes/integration.md [hunk 6]
```diff
@@ -806,10 +781,10 @@ access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource
 
 Required | Name             | Type      | Description                                                                              | Default Value
 ---------|------------------|-----------|------------------------------------------------------------------------------------------|--------------
-yep      | from_datetime    | date_time | The date_time from which you want the schedules                                          |
+yep      | from_datetime    | [iso-date-time](#iso-date-time) | The date_time from which you want the schedules                                          |
 nop      | duration         | int       | Maximum duration in seconds between from_datetime and the retrieved datetimes.           | 86400
 nop      | max_date_times   | int       | Maximum number of columns per schedule.                                                  |
-nop      | data_freshness   | enum      | Define the freshness of data to use<br><ul><li>real_time</li><li>base_schedule</li></ul> | base_schedule
+nop      | data_freshness   | enum      | Define the freshness of data to use<br><ul><li>realtime</li><li>base_schedule</li></ul> | base_schedule
```
Comment (pbougue) [inline documentation/slate/source/includes/integration.md]: I think you missed the `forbidden_uris[]` parameter here (https://github.com/CanalTP/navitia/commit/bae95b4ce6f46cc9c46bd4c8e014771fb1b684d5)

**Comment by pbougue** [comment]
:+1:

File: documentation/slate/source/includes/index.md [hunk 7]
```diff
@@ -62,52 +73,49 @@ This chapter shows some usages with the minimal required arguments. However, thi
...
 The arguments are the following:
 
 
 * ``from=-122.4752;37.80826``
 * ``to=-122.402770;37.794682``
-* ``datetime=20140118T0800``
-
-Hence, the complete URL: <http://api.navitia.io/v1/journeys?from=-122.4752;37.80826&to=-122.402770;37.794682&datetime=20140618T0800>.
-
+Hence, the complete URL: <http://api.navitia.io/v1/journeys?from=-122.4752;37.80826&to=-122.402770;37.794682&datetime=20160422T0800>.
```
Comment (pbougue) [inline documentation/slate/source/includes/index.md]: I would remove the datetime in the url here.

**Pull request merged**